entrypoint.sh

```diff
@@ -15,7 +15,7 @@ start() {
     fi
 
     cp ${TMP_DIR}/spigot-server.jar ${OPT_DIR}/spigot-server.jar
-    (screen -DmS spigot java -Xms2048M -Xmx2048M -jar ${OPT_DIR}/spigot-server.jar ${spigot_args}) &
+    (screen -DmS spigot java -Xms${MIN_MEMORY} -Xmx${MAX_MEMORY} -jar ${OPT_DIR}/spigot-server.jar ${spigot_args}) &
```

Comment:
代入されていないときはデフォルトで 2048M でいいと思います。

POSIX Shell の Parameter Expansion ではデフォルト値を利用できるのでこれを使えば実装できるはずです。
https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02

```suggestion
    (screen -DmS spigot java -Xms${MIN_MEMORY:-2048M} -Xmx${MAX_MEMORY:-2048M} -jar ${OPT_DIR}/spigot-server.jar ${spigot_args}) &
```

Reply:
ありがとう
知らなかった